scipy/optimize/_differentialevolution.py

```diff
@@ -294,6 +295,53 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
     iteration, rather than multiple times for all the population members; the
     interpreter overhead is reduced.
 
+    An outline of the strategies and their stengths and weaknesses is given by 
+    Qiang and Mitchell (2014) [8]_
+
+    ..math::
+            DE/rand/1 : v_i = x_{r_1} + F_{xc}(x_{r_2} − x_{r_3})
+
+            DE/rand/2 : v_i = x_{r_1} + F_{xc}(x_{r_2} − x_{r_3}) 
```

Comment:
On L260 we refer to population members as $b_0$, i.e. generalising to $b_x$. Can you change `x_{r_2}` to `b_{r_2}` in all these equations?

Reply:
happy to change this, however we run into problems with the vector x_b which is supposed to represent the "best vector" and in the line260 notation we refer to the best vector as b_0. Do you think we should refer to best vector as b_b?

Alternatively we could change the notation on line 260 to x_b and I can change x_{r_2} --> population[rand1] for the remaining documentation to match the notation on line 260.

Reply:
The best vector is always kept in the first element of the population, b[0], $b_0$. Thus this equation should be rewritten:
```
b' = b_{r_1} + mutation * (b_{r_2} − b_{r_3}) 
```

Reply:
I understand. Great, I will fix.